jaeger-core/src/main/java/io/jaegertracing/internal/MDcScope.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2018, The Jaeger Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.jaegertracing.internal;
+
+import io.opentracing.Scope;
+import io.opentracing.Span;
+import org.slf4j.MDC;
+
+public class MDcScope implements Scope {
+  private static final String TRACE_ID = "traceId";
+  private static final String SPAN_ID = "spanId";
+  private static final String SAMPLED = "sampled";
```

Comment:
I think these names need to be configurable. Some organizations already have established patterns for logging similar data from other languages.

I think it will require a Builder for the ScopeManager.

Reply:
The names in log messages are configurable via pattern e.g.
```
format=%d{HH:mm:ss} %-5p traceId=%X{traceId}, spanId=%X{spanId}, 
```

the question more is whether our keys could collide with the other keys. I think the risk is pretty minimal.

Reply:
I added a builder to the MDCscopeManger you can change the fields or use the default.